### DISCUSSION TITLE
Azerothcore Modules Help

### DISCUSSION THREAD

**Discussion by Nukems91** [azerothcore/azerothcore-wotlk#11607] [General]
Hey guys,

I recently created my own WoW Server using Azerothcore and Docker the server is running perfectly no issues however I can't for the life of me understand how to install modules.... 

Just for an example I have installed the mod learn spells master inside the module folder, I have used Cmake to config and generate with no issues than I reset the server but once I log in there is no mod installed... I've been at it all day and just super confused... any help woudl be appreciative. 

cheers,

**Comment by Kitzunu** [comment]
you need to recompile after cmake

**Reply by Nukems91** [reply]
> you need to recompile after cmake

Sorry mate I am literally learning everything the last 2 days how do I recompile?

**Reply by Kitzunu** [reply]
the same way you did it the first time. I am no docker expert, but it would be the step before you are able to launch your world and auth server. Which usually takes a few minutes to build
